src/inference/src/threading/ie_istreams_executor.cpp

```diff
@@ -162,8 +175,12 @@ void IStreamsExecutor::Config::SetConfig(const std::string& key, const std::stri
         } else if (streams == ov::streams::AUTO) {
             // bare minimum of streams (that evenly divides available number of cores)
             _streams = GetDefaultNumStreams();
+            _streams = LimitHybridStreams(_streams, _threads);
```

Comment:
Please do not impact CORES affinity in this PR.

Reply:
I added the Hybrid Aware conditon.